packages/next-swc/crates/next-core/js/src/ipc/server.ts

```diff
@@ -113,7 +114,13 @@ export function makeRequest(
     // Otherwise Node.js waits for the first chunk of data to be written before sending the request.
     clientRequest.flushHeaders();
 
+    if (body) {
+      clientRequest.write(body);
+    }
+
     clientRequest.once("response", responseListener);
     clientRequest.once("error", clientResponseErrorListener);
+
+    clientRequest.end();
```

Comment:
You might even want to wait for the connection buffer to be flushed?

```suggestion
    await new Promise((resolve) => clientRequest.end(() => resolve()));
```

Reply:
I don't think that's necessary, this is also not an async function